src/plugins/data/server/ui_settings.ts

```diff
@@ -762,5 +762,19 @@ export function getUiSettings(): Record<string, UiSettingsParams<unknown>> {
       }),
       schema: schema.arrayOf(schema.string()),
     },
+    [UI_SETTINGS.SEARCH_USE_FIELDS]: {
+      name: i18n.translate('data.advancedSettings.searchUseFieldsTitle', {
+        defaultMessage: 'Include all fields in search request',
```

Comment:
this setting is to include all fields, would `search:includeAllFields` be better than `search:useFields`?

Reply:
My thinking was that previously, we weren't using the `fields` field in the request body. Turning on this property "uses" `fields`. I think in this case, the title itself could be something different.

Reply:
I like this suggestion